tests/general/InvalidLinkBearTest.py

```diff
@@ -149,6 +149,9 @@ def test_link_ignore_regex(self):
         ignored_URLs = """
         http://sub.example.com
         http://sub.example.com/something
+        http://www.%s.com
+        http://www.%d.com
+        http://www.%f.com
```

Comment:
Should I modify regex again so that ``http://www.google.com/hello%20world`` is ignored and the test pass ?